lib/kaffe/group_member/manager/group_member_supervisor.ex

```diff
@@ -0,0 +1,34 @@
+defmodule Kaffe.GroupMemberSupervisor do
+  @moduledoc """
+  The top-level supervisor for group members and subscribers.
+  """
+
+  use Supervisor
+
+  require Logger
+
+  def start_link do
+    Supervisor.start_link(__MODULE__, :ok)
+  end
+
+  def start_worker_supervisor(supervisor_pid) do
+    Supervisor.start_child(supervisor_pid, supervisor(Kaffe.WorkerSupervisor, []))
+  end
+
+  def start_group_member(supervisor_pid, subscriber_name, consumer_group,
+      worker_manager_pid, topic, configured_offset) do
+    Supervisor.start_child(supervisor_pid, worker(Kaffe.GroupMember,
+      [subscriber_name, consumer_group, worker_manager_pid, topic, configured_offset], id: topic))
```

Comment:
When reading about supervisors, I understand that it's idiomatic to try to keep them relatively simple and instead have a manager that handles the logic. But I'm open about it.